### DISCUSSION TITLE
Filtering Project Board by Pull Request Reviewer

### DISCUSSION THREAD

**Discussion by jeffpaul** [community/community#5648] [Projects and Issues]
There are several options for filtering a project board (see: https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/filtering-cards-on-a-project-board) however none of those allow you to filter to see what items you (or someone on your team) are set as a reviewer for.  Having that added would be a great enhancement to allow team members to build a specific set of filters to view just their open issues/PRs/items for review.  As best I can tell from searching there is no good way to do this right now, but would love to be proven wrong and shown how to do so.

**Comment by erikr** [comment]
Yes, seconded! This would be extremely helpful.
